bin/ChakraCore/ChakraCore.vcxproj

```diff
@@ -52,13 +52,9 @@
         Rpcrt4.lib;
         $(ChakraCommonLinkDependencies)
       </AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(IntlICU)'=='true'">
-        %(AdditionalDependencies);
-        <!-- TODO cleanup: we do need to link to ICU libraries in ChakraCore, but we probably will not need all of them -->
-        $(IcuLibDir)\icutools.lib;
-        $(IcuLibDir)\icui18n.lib;
-        <!--$(IcuLibDir)\icudata.lib;-->
-        <!--$(IcuLibDir)\icuucx.lib;-->
+      <AdditionalDependencies Condition="'$(UseICU)'=='true' AND '$(ChakraICU)'!='false'">
```

Comment:
This should be `AND '$(ChakraICU)'=='false'`, since this only needs to happen when we are using arbitrary ICU libraries.